ionic2/src/models/mob.ts

```diff
@@ -4,7 +4,8 @@ export class Mob extends POI {
   clusterId: number;
   tweets: MobTweet[];
   timestamp: number;
-
+  coordinates: [number, number]; // [longitude, latitude]
+  isMob: boolean;
```

Comment:
Well this is just how the mob object looks like https://github.com/PokemonGoers/PokeMap-2/issues/32#issuecomment-246179575